TicketSwap/solar
# Add support to optional chaining operator

# Description

Add support to optional chaining operator.
Fixes #1033 

## Changes

- Added babel plugin: https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining

## How to test

- Checkout this branch
- `$ yarn build`
- build should be done without errors
